commands/update.js

```diff
@@ -0,0 +1,58 @@
+const exec = require('child_process').exec;
+
+module.exports = {
+  describe: 'Update the SparkPost CLI',
+  action: function(callback) {
+    let packageName = require('../package.json').name;
+
+    isFromNPM(packageName, function(fromNPM) {
```

Comment:
Maybe use a standard `function(err, fromNPM) {}` callback style so you can pass errors back up the chain?